### ISSUE TITLE
cells with multi-lines are treated as different rows

### ISSUE THREAD

**Issue by vishnu41** [tabulapdf/tabula-java#238]

**Comment by rakshitcgupta** [comment]
Yeah, in the stream files, the multilines is treated as different rows as there is no way of knowing whether the next line is in the same cell. But these are properly merged for the lattice files.

**Comment by vishnu41** [comment]
Is there anyway to use both -g and -l ?
